probe_scraper/parsers/repositories.py

```diff
@@ -175,6 +175,8 @@ def _v2_to_v1(self, filename):
             app.pop("app_name")
             app.pop("canonical_app_name", None)
             app.pop("bq_dataset_family")
+            app["description"] = app["app_description"]
+            app.pop("app_description", None)
```

Comment:
```suggestion
            app_description = app.pop("app_description", None)
            app["description"] = app.get("description", app_description)
```

To maintain current behavior for v1, I think we want to maintain "description" if it exists at the listing level, and only insert app_description when "description" is otherwise absent.

Reply:
Right, can't believe I missed that!